task_DEV_2/task_DEV_2/Program.cs

```diff
@@ -4,46 +4,57 @@ namespace task_DEV_2
 {
     class Program
     {
-        static void Main(string[] args)
+        static void InputArrayElements(int[] inputNumbers, int arrayLength)
```

Comment:
Please change the name of the method and add return value.
static int[] CreateNumbersArray(int arrayLength)

Reply:
done